poreCov.nf

```diff
@@ -466,12 +466,12 @@ ${c_yellow}Parameters - Basecalling  (optional)${c_reset}
 
 ${c_yellow}Parameters - SARS-CoV-2 genome reconstruction (optional)${c_reset}
     --primerV       Supported primer variants - choose one [default: ${params.primerV}]
-                        ${c_dim}ARTIC:${c_reset} V1, V2, V3, V4 
+                        ${c_dim}ARTIC:${c_reset} V1, V2, V3, V4, V4.1
```

Comment:
but here I have to use V4-1, or? as the parameter

Reply:
I renamed it. I typed V4-1 and it was ugly...

Reply:
:D okay so now one can use `--primerV V4.1` ?

Reply:
I checked - `{params.primerV}` is always used as it is

Reply:
Yes, `--primerV V4.1` should work :)